cpp/ql/src/semmle/code/cpp/ir/dataflow/DefaultTaintTracking.qll

```diff
@@ -90,20 +92,20 @@ private class ToGlobalVarTaintTrackingCfg extends DataFlow::Configuration {
     sink.asVariable() instanceof GlobalOrNamespaceVariable
   }
 
-  override predicate isAdditionalFlowStep(DataFlow::Node n1, DataFlow::Node n2) {
+  override predicate isAdditionalTaintStep(DataFlow::Node n1, DataFlow::Node n2) {
     commonTaintStep(n1, n2)
```

Comment:
I expected that all calls to `commonTaintStep` could be removed when you converted the last configurations to use `TaintTracking`. Or at least the body of `commonTaintStep` could cut down to contain just some subset of what's now in `AdjustedConfiguration::isAdditionalTaintStep`.

Reply:
Oh, good point. They should all be removed, indeed.

Reply:
Fixed in 3914a93504bb92bb514fe74cb609662f683f290b.

Reply:
:heart_eyes: